canonical/canonical.com
# WD-8721 - create /solutions/infrastructure page

## Done

Create infrastructure page

Note: for testing purposes the link for successful form submission will be changed to the demo link. Will be updated before merge

## QA

- [demo link](https://canonical-com-1184.demos.haus/solutions/infrastructure)
- [copy doc](https://docs.google.com/document/d/1xbuIA53HKrMatjuiNQwf1ORwLvNXFEF0UqGTbB6sXL0/edit)
- [figma](https://www.figma.com/file/Cova41wSG3JIq9NPNZnC50/Canonical.com%2Fsolutions%2Finfrastructure?type=design&node-id=0-1&mode=design&t=eYW5H4A3x6cEa661-0)
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page and it's content is according to copy doc

## Issue / Card
[WD-8721](https://warthogs.atlassian.net/browse/WD-8721)

Fixes #

## Screenshots


[WD-8721]: https://warthogs.atlassian.net/browse/WD-8721?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
